include/exiv2/datasets.hpp

```diff
@@ -132,12 +132,10 @@ class EXIV2API IptcDataSets {
   static constexpr uint16_t Preview = 202;
   //@}
 
-  //! Prevent construction: not implemented.
-  IptcDataSets() = delete;
```

Comment:
What's the reasoning to allow the creation of this class now? This change looks out of the scope of the commit.

Reply:
A default constructor does not get generated here. I removed it as it's a bit misleading.

Reply:
Oh god, you're totally right. I am too far from coding lately and I forget these details about C++ 😩

Reply:
I think leaving the explicit `= delete` doesn't hurt though and helps a lot when reading the code as one then doesn't need to worry about knowing the details of when special member functions are implicitly declared...

Reply:
I also think that sometimes being explicit about certain things is better for code readability, but it is difficult to define where to draw the line. On the other hand, less code is almost always the way to go for me. In this case, I think it is fine to remove those lines, if somebody try to create an instance of the class with the default constructor the compiler will complain anyways.